### ISSUE TITLE
Tawk.to issue/navigator.plugins undefined

### ISSUE THREAD

**Issue by edde746** [BuilderIO/partytown#270]
- [ ] I am interested in helping provide a fix!

**Describe the bug**
navigator.plugins is undefined causing the script not to load

**To Reproduce**
[See repl](https://replit.com/@edde746/Tawkto-Issue), check console

**Expected behavior**
Script should load without error

**Partytown version**
Latest?

**Screenshots**
![Error](https://i.snipboard.io/EqC3xG.jpg)

**Additional context**
Seems like the issue is that `navigator.plugins` isn't available in the service worker.

**Comment by itxch** [comment]
Seems like the issue stems from the fact that `navigator` is created based on WebWorkerNavigator, not `window.navigator`.
So it only has these methods: https://developer.mozilla.org/en-US/docs/Web/API/WorkerNavigator
For other methods - such as `navigator.plugins` it'd probably need a custom implementation

**Comment by vip30** [comment]
The same issue happens in chartbeat_video.js

**Comment by solamichealolawale** [comment]
What's the update on this? Are we suppose to load in the main thread?

**Comment by anhle82** [comment]
I have similar issue with another 3rd party script that requires navigator.vendor. Has anyone figured out how to solve this navigator issue for web worker?

**Comment by cornedor** [comment]
I'm not sure what Tawk is using plugins for, but as a workaround I've added this line to the Tawk snippet:

```
window.navigator.plugins = navigator.pdfViewerEnabled ? ['PDF Viewer'] : [];

```
This seems to be the only usecase of navigator.plugins: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/plugins

**Comment by gioboa** [comment]
Closed by #539
